### PR TITLE
Changed colors and example images

### PR DESCRIPTION
**Pull request by IbrahimKhan07**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,50 +1,50 @@
 # QB Styles
 
 [![License](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
-[![Python Version](https://img.shields.io/pypi/pyversions/qbstyles.svg)](https://pypi.org/project/qbstyles/)
-[![PyPI version](https://badge.fury.io/py/qbstyles.svg)](https://pypi.org/project/qbstyles/)
+[![Python Version](https://img.shields.io/pypi/pyversions/sneeifstyles.svg)](https://pypi.org/project/sneeifstyles/)
+[![PyPI version](https://badge.fury.io/py/sneeifstyles.svg)](https://pypi.org/project/sneeifstyles/)
```
Comment (AJarman) [inline README.md:5]: we are not publishing on pypi, please remove links

File: README.md [hunk 2]
```diff
@@ -1,50 +1,50 @@
...
 
 ## How do I install QB Styles?
 
-`qbstyles` is a Python package. To install it, simply run:
+`sneeifstyles` is a Python package. To install it, simply run:
 
 
 ```bash
-pip install qbstyles
+pip install sneeifstyles
```
Comment (AJarman) [inline README.md:20]: change to this: 

pip install git+https://github.com/SNEE-ICS/sneeifstyles.git

File: README.md [hunk 3]
```diff
@@ -1,50 +1,50 @@
...
 
 mpl_style(dark=False)
 ```
 
 ### How do I use QB Styles in Jupyter Notebooks?
 
-> ⚠️ Please make sure you run `from qbstyles import mpl_style` and `mpl_style()` in **different cells** as shown below. See [this issue](https://github.com/jupyter/notebook/issues/3691) for more details.
+> ⚠️ Please make sure you run `from sneeifstyles import mpl_style` and `mpl_style()` in **different cells** as shown below. See [this issue](https://github.com/jupyter/notebook/issues/3691) for more details.
 
 ```python
```
Comment (AJarman) [inline README.md:45]: should just work now in the same cell? please check and remove issue link if working

File: README.md [hunk 4]
```diff
@@ -85,23 +85,23 @@ def plot(dark):
 plot(dark=True)
 ```
 
-![png](https://github.com/quantumblacklabs/qbstyles/raw/master/examples/output_6_0.png?raw=true)
+![png](https://github.com/quantumblacklabs/sneeifstyles/raw/master/examples/output_6_0.png?raw=true)
```
Comment (AJarman) [inline README.md:88]: fix URL

File: README.md [hunk 5]
```diff
@@ -85,23 +85,23 @@ def plot(dark):
...
 
 ```python
 plot(dark=False)
 ```
 
-![png](https://github.com/quantumblacklabs/qbstyles/raw/master/examples/output_7_0.png?raw=true)
+![png](https://github.com/quantumblacklabs/sneeifstyles/raw/master/examples/output_7_0.png?raw=true)
```
Comment (AJarman) [inline README.md:94]: fix URL

File: sneeifstyles/colours.py [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+]
+
+CORE_BLUE_CYCLE = [
+    #TODO add blue colours
+]
+
+CORE_YELLOW_CYCLE = [ 
+    #TODO add yellow colours
+]
+
```
Comment (AJarman) [inline sneeifstyles/colours.py:33]: need to add the colours as per #TODO tags, use the document provided by elis

**Review by AJarman — CHANGES_REQUESTED** [review]
Please review changes and implement, thanks

File: README.md (hunk 1 above)
Comment (IbrahimKhan07) [reply README.md]: done

File: README.md [hunk 7]
```diff
@@ -1,50 +1,50 @@
...
+[![Python Version](https://img.shields.io/pypi/pyversions/sneeifstyles.svg)](https://pypi.org/project/sneeifstyles/)
+[![PyPI version](https://badge.fury.io/py/sneeifstyles.svg)](https://pypi.org/project/sneeifstyles/)
 [![Code Style: Black](https://img.shields.io/badge/code%20style-black-black.svg)](https://github.com/ambv/black)
 
 QB Styles is a python package with a light and a dark [`matplotlib`](https://github.com/matplotlib/matplotlib) style.
 
 Dark style | Light style
 |-----------|----------- |
-| ![Line plot](https://github.com/quantumblacklabs/qbstyles/raw/master/examples/line.png?raw=true "Line plot") | ![Distribution plot](https://github.com/quantumblacklabs/qbstyles/raw/master/examples/distribution_light.png?raw=true "Distribution plot") |
+| ![Line plot](https://github.com/quantumblacklabs/sneeifstyles/raw/master/examples/line.png?raw=true "Line plot") | ![Distribution plot](https://github.com/quantumblacklabs/sneeifstyles/raw/master/examples/distribution_light.png?raw=true "Distribution plot") |
```
[the remark this answers is not in the document]
Comment (IbrahimKhan07) [reply README.md]: changed

File: README.md (hunk 2 above)
Comment (IbrahimKhan07) [reply README.md]: changed

File: README.md (hunk 3 above)
Comment (IbrahimKhan07) [reply README.md:44]: It works in 1 cell, have checked and pushed changes

File: README.md (hunk 4 above)
Comment (IbrahimKhan07) [reply README.md]: fixed

File: README.md (hunk 5 above)
Comment (IbrahimKhan07) [reply README.md]: fixed

File: README.md [hunk 8]
```diff
@@ -85,23 +85,23 @@ def plot(dark):
...
 
 To do so, create a file similar to the above files at the root of your project, and apply it after the `qbstyle` as follows:
 
 ```python
 import matplotlib.pyplot as plt
-from qbstyles import mpl_style
+from sneeifstyles import mpl_style
 
 mpl_style()
 plt.style.use('./your-style.mplstyle')
```
[the remark this answers is not in the document]
Comment (IbrahimKhan07) [reply README.md]: removed

File: sneeifstyles/colours.py (hunk 6 above)
Comment (IbrahimKhan07) [reply sneeifstyles/colours.py]: added

**Pull request merged**